lib/migration.pm

```diff
@@ -207,4 +209,19 @@ sub set_scc_proxy_url {
     save_screenshot;
 }
 
+# Enable ZYPP_SINGLE_RPMTRANS=1 if the OS have
+# libzypp-17.28.4 and zypper-1.14.49 or higher version installed
+sub set_zypp_single_rpmtrans {
+    my $zypper_ver = script_output q(rpm -ivq zypper | awk -F\- '{print $2}');
```

Comment:
Suggest changes:
 my $current_ver = script_output("rpm -q --qf '%{version}\n' x3270");

Reply:
> Suggest changes: my $current_ver = script_output("rpm -q --qf '%{version}\n' x3270");
Thanks a lot, but it's out of my wit to understand that.

Reply:
I think rpm -q --qf '%{version}\n' pkg_name can give "more" correct pkg version here.

e.g. your pkg version is 1.2.3-4.5, your current code may give out 1.2.3, correct?

Reply:
now use setting ZYPP_SINGLE_RPMTRANS. so this is not a question now. thanks

Reply:
Sure, up to you